config/dm_config/nnf-dm-config.yaml

```diff
@@ -1,6 +1,30 @@
+# Each profile is capable of providing different configurations for data movement.
+# Note: For now, only the default profile is supported.
 profiles:
+
+  # Default profile that is used for all data movement activity.
   default:
+      # The number of slots to be use in the MPI hostfile. A value less than 1 disables the use
+      # of slots in the hostfile.
       slots: 8
+
+      # The number of max_slots to be use in the MPI hostfile. A value less than 1 disables the use
+      # of max_slots in the hostfile.
       maxSlots: 0
+
+      # The full command to execute data movement. $VARS are replaced by the nnf software. Available
+      # $VARS:
+      #   HOSTFILE: hostfile that is created and used for mpirun. Contains a list of hosts and the
+      #             slots/max_slots for each host. This hostfile is created at `/tmp/<dm-name>/hostfile`
+      #   UID: User ID that is inherited from the Workflow
+      #   GID: Group ID that is inherited from the Workflow
+      #   SRC: source for the data movement
```

Comment:
HOSTFILE, SRC and DEST are controlled by Rabbit software, right?  Say that here.

Reply:
All of them are. I do mention that on line 15.